pallets/protos/src/dummy_data.rs

```diff
@@ -281,24 +263,24 @@ impl DummyData {
 		};
 
 		let contracts = vec![String::from("0x8a819F380ff18240B5c11010285dF63419bdb2d5")];
-		let contract = Address::from_str(&contracts[0].as_str()[2..]).map_err(|_| "Invalid response - invalid sender").unwrap();
+		let contract = Address::from_str(&contracts[0].as_str()[2..])
+			.map_err(|_| "Invalid response - invalid sender")
+			.unwrap();
```

Comment:
Maybe better to bubble up errors, in case of any using, `.map_err(|_| "Invalid response - invalid sender")?` instead of `.map_err(|_| "Invalid response - invalid sender").unwrap()` ?.

Reply:
I dunno what's this. Oh dummy data? Guess doesn't matter 🤔

Reply:
oh didn't see it was dummy data. Sure.